secret-script.py

```diff
@@ -1,14 +1,3 @@
-# Lint as: python3
```

Comment:
Allow this to be directly executable via whichever `python3` is available first in PATH
```suggestion
#!/usr/bin/env python3
```

Reply:
Amazed.
Fixed.

Reply:
Ah right, `./script.py` won't work until you `chmod +x` this file